workshops/base_views.py

```diff
@@ -206,3 +206,19 @@ def get_success_url(self):
             return next_url
         else:
             return default_url
+
+
+class PrepopulationSupportMixin:
+
+    def get_initial(self):
+        return self.request.GET.dict()
+
+    def get_form(self, *args, **kwargs):
+        """Disable fields that are pre-populated."""
+        form = super().get_form(*args, **kwargs)
+        for field in self.get_initial():
+            try:
+                form.fields[field].disabled = True
```

Comment:
I agree this is an issue with `django-selectable`, but it'd be very confusing to have an enabled field that is ignored finally. IMO fields should be left enabled or made 100% disabled.

> The issue was fixed at django-selectable #170

It seems that this fix wasn't released yet (django-selectable 0.9 is from 2014), so we can't fix it by upgrading `django-selectable`. So, if we want to make them disabled, we need a workaround like [this one](https://github.com/mlavin/django-selectable/issues/163). Alternatively, we can leave prepopulated fields enabled and disable them when the fix is released.